lightning/src/ln/channel.rs

```diff
@@ -2187,24 +2195,23 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 						},
 					}
 					if err.is_some() {
-						self.holding_cell_htlc_updates.push(htlc_update);
 						if let Some(ChannelError::Ignore(_)) = err {
 							// If we failed to add the HTLC, but got an Ignore error, we should
 							// still send the new commitment_signed, so reset the err to None.
 							err = None;
+						} else {
+							self.holding_cell_htlc_updates.push(htlc_update);
```

Comment:
Hmm, I think its fine to move this here, but it would be nice to add a comment why its fine to drop ChannelError::Ignore  responses from get_update_fail_htlc and get_update_fulfill_htlc.

Reply:
Hope I interpreted this correctly